### ISSUE TITLE
MySQL and Postgres missing dependencies

### ISSUE THREAD

**Issue by kmlucy** [netdata/netdata#9714]
##### Bug report summary
I updated to v1.24, and my collectors for MySQL and Postgres are no longer working.

Here are the relevant logs:
```
2020-08-11 07:12:14: python.d ERROR: postgres[postgres] : 'python-psycopg2' package is needed to use postgres module
2020-08-11 07:12:14: python.d INFO: plugin[main] : postgres[postgres] : check failed
...
2020-08-11 07:12:14: python.d ERROR: mysql[mariadb] : MySQLdb or PyMySQL module is needed to use mysql.chart.py plugin
2020-08-11 07:12:14: python.d INFO: plugin[main] : mysql[mariadb] : check failed
```

##### OS / Environment
I am running in the official Docker container.

##### Netdata version
`netdata v1.24.0-9-g6caa2b65`
 
##### Component Name
collectors/python.d.plugin/postgres
collectors/python.d.plugin/mysql

##### Steps To Reproduce
1. Pull latest Docker image
2. Start container with MySQL and Postgres configurations

##### Expected behavior
Collectors to work

**Comment by prologic** [comment]
@ilyam8 Can you please look into ☝️ -- When I made the changes to reduce the size of our Docker images, part of my "Test Plan" was to ensure the Python interpreter on the image was able to `import <module>` as required. If the collectors themselves are looking for something else, then that's a different issue.

**Comment by ilyam8** [comment]
> part of my "Test Plan" was to ensure the Python interpreter on the image was able to import <module>

error
> 2020-08-11 07:12:14: python.d ERROR: postgres[postgres] : 'python-psycopg2' package is needed to use postgres module


https://github.com/netdata/netdata/blob/77714ecb5a9dce56c097a588e87f97980916a9a4/collectors/python.d.plugin/postgres/postgres.chart.py#L8-L16

https://github.com/netdata/netdata/blob/77714ecb5a9dce56c097a588e87f97980916a9a4/collectors/python.d.plugin/postgres/postgres.chart.py#L972-L975

---

tl dr: cant import `psycopg2`

**Comment by ilyam8** [comment]
Oh, it is `psycopg2` - python2

**Comment by ilyam8** [comment]
```cmd
0 ~ $ docker exec -it heuristic_northcutt bash
bash-5.0# netdata -v
netdata v1.24.0-24-gdc1fa08a

bash-5.0# su -s /bin/bash netdata
bash-5.0$ whoami
netdata


bash-5.0$ /usr/libexec/netdata/plugins.d/python.d.plugin postgres
2020-08-13 12:04:46: python.d INFO: plugin[main] : using python v3
2020-08-13 12:04:46: python.d WARNING: plugin[main] : 'pythond-jobs-statuses.json' was not found
2020-08-13 12:04:46: python.d INFO: plugin[main] : [postgres] built 4 job(s) configs
2020-08-13 12:04:46: python.d ERROR: postgres[socket] : 'python-psycopg2' package is needed to use postgres module
2020-08-13 12:04:46: python.d INFO: plugin[main] : postgres[socket] : check failed
2020-08-13 12:04:46: python.d ERROR: postgres[tcp] : 'python-psycopg2' package is needed to use postgres module
2020-08-13 12:04:46: python.d INFO: plugin[main] : postgres[tcp] : check failed
2020-08-13 12:04:46: python.d ERROR: postgres[tcpipv4] : 'python-psycopg2' package is needed to use postgres module
2020-08-13 12:04:46: python.d INFO: plugin[main] : postgres[tcpipv4] : check failed
2020-08-13 12:04:46: python.d ERROR: postgres[tcpipv6] : 'python-psycopg2' package is needed to use postgres module
2020-08-13 12:04:46: python.d INFO: plugin[main] : postgres[tcpipv6] : check failed
2020-08-13 12:04:46: python.d INFO: plugin[main] : no jobs to serve
2020-08-13 12:04:46: python.d INFO: plugin[main] : exiting from main...

bash-5.0$ python3 -c "import psycopg2"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'psycopg2'
```

**Comment by prologic** [comment]
Wait... There's something odd going on here. I tested this damnit :)

**Comment by prologic** [comment]
https://github.com/netdata/helper-images/pull/84 is merged

but the dependent PR https://github.com/netdata/netdata/pull/9669 is not

:/ 

Merging now...

**Comment by kmlucy** [comment]
Postgres is fixed with `v1.24.0-85-g07e67c4b`, but I still have the same logs for MySQL:
```
2020-08-20 07:23:06: python.d ERROR: mysql[mariadb] : MySQLdb or PyMySQL module is needed to use mysql.chart.py plugin
2020-08-20 07:23:06: python.d INFO: plugin[main] : mysql[mariadb] : check failed
```

**Comment by kmlucy** [comment]
I don't use any of the others, but a lot of dependency modules seem to be missing, though I don't know if this is new or not:

```
$ cat logs | grep -A 1 'is needed'
2020-08-20 07:23:06: python.d ERROR: beanstalk[beanstalk] : 'beanstalkc' module is needed to use beanstalk.chart.py
2020-08-20 07:23:06: python.d INFO: plugin[main] : beanstalk[beanstalk] : check failed
--
2020-08-20 07:23:06: python.d ERROR: ceph[ceph] : rados module is needed to use ceph.chart.py
2020-08-20 07:23:06: python.d INFO: plugin[main] : ceph[ceph] : check failed
--
2020-08-20 07:23:06: python.d ERROR: dns_query_time[dns_query_time] : 'python-dnspython' package is needed to use dns_query_time.chart.py
2020-08-20 07:23:06: python.d INFO: plugin[main] : dns_query_time[dns_query_time] : check failed
--
2020-08-20 07:23:06: python.d ERROR: dockerd[local] : 'docker' package is needed to use dockerd module
2020-08-20 07:23:06: python.d INFO: plugin[main] : dockerd[local] : check failed
--
2020-08-20 07:23:06: python.d ERROR: mysql[mariadb] : MySQLdb or PyMySQL module is needed to use mysql.chart.py plugin
2020-08-20 07:23:06: python.d INFO: plugin[main] : mysql[mariadb] : check failed
--
2020-08-20 07:23:06: python.d ERROR: openldap[openldap] : 'python-ldap' package is needed
2020-08-20 07:23:06: python.d INFO: plugin[main] : openldap[openldap] : check failed
2020-08-20 07:23:06: python.d ERROR: oracledb[oracledb] : 'cx_Oracle' package is needed to use oracledb module
2020-08-20 07:23:06: python.d INFO: plugin[main] : oracledb[oracledb] : check failed
--
2020-08-20 07:23:06: python.d ERROR: proxysql[tcp] : MySQLdb or PyMySQL module is needed to use mysql.chart.py plugin
2020-08-20 07:23:06: python.d INFO: plugin[main] : proxysql[tcp] : check failed
2020-08-20 07:23:06: python.d ERROR: proxysql[tcpipv4] : MySQLdb or PyMySQL module is needed to use mysql.chart.py plugin
2020-08-20 07:23:06: python.d INFO: plugin[main] : proxysql[tcpipv4] : check failed
2020-08-20 07:23:06: python.d ERROR: proxysql[tcpipv6] : MySQLdb or PyMySQL module is needed to use mysql.chart.py plugin
2020-08-20 07:23:06: python.d INFO: plugin[main] : proxysql[tcpipv6] : check failed
2020-08-20 07:23:06: python.d ERROR: proxysql[tcp_admin] : MySQLdb or PyMySQL module is needed to use mysql.chart.py plugin
2020-08-20 07:23:06: python.d INFO: plugin[main] : proxysql[tcp_admin] : check failed
2020-08-20 07:23:06: python.d ERROR: proxysql[tcpipv4_admin] : MySQLdb or PyMySQL module is needed to use mysql.chart.py plugin
2020-08-20 07:23:06: python.d INFO: plugin[main] : proxysql[tcpipv4_admin] : check failed
2020-08-20 07:23:06: python.d ERROR: proxysql[tcpipv6_admin] : MySQLdb or PyMySQL module is needed to use mysql.chart.py plugin
2020-08-20 07:23:06: python.d INFO: plugin[main] : proxysql[tcpipv6_admin] : check failed
--
2020-08-20 07:23:06: python.d ERROR: rethinkdbs[local] : "rethinkdb" module is needed to use rethinkdbs.py
2020-08-20 07:23:06: python.d INFO: plugin[main] : rethinkdbs[local] : check failed
```

**Comment by ilyam8** [comment]
> but I still have the same logs for MySQL

Yeah, i suggest to use [go mysql module](https://github.com/netdata/go.d.plugin/tree/master/modules/mysql#mysql-monitoring-with-netdata)

What we plan to do with all those deps - rewrite modules in go, so we dont need any external libs

**Comment by ilyam8** [comment]
have go versions:

- python-dnspython
- MySQLdb

have no:

- beanstalkc
- rados
- docker
- python-ldap
- cx_Oracle
- rethinkdb

**Comment by kmlucy** [comment]
I wasn't aware to the MySQL go module. I've transitioned my config over and am okay closing out this issue unless you'd rather keep it open.

**Comment by ilyam8** [comment]
postgress is added, mysql is wontfix i think, so i am closing the issue
